src/core_plugins/kibana/public/discover/components/field_chooser/discover_field.html

```diff
@@ -16,7 +16,7 @@
         ng-click="toggleDisplay(field)"
         ng-class="::field.display ? 'kuiButton--danger' : 'kuiButton--primary'"
         ng-bind="::field.display ? 'remove' : 'add'"
-        class="kuiButton kuiButton--small kuiButton--primary"
+        class="kuiButton kuiButton--small kuiButton--primary discover-field-toggle"
```

Comment:
Instead of embedding this logic in the CSS, can we move it here so it's more discoverable:

```html
<li
  class="sidebar-item"
  attr-field="{{::field.name}}"
>
  <div
    data-test-subj="field-{{::field.name}}"
    ng-click="toggleDetails(field)"
    class="sidebar-item-title discover-sidebar-item"
  >
    <field-name
      class="discover-sidebar-item-label"
      field="field"
    ></field-name>

    <!-- _source field can't be removed -->
    <div
      class="discover-sidebar-item-actions"
      ng-if="field.name !== '_source'"
    >
      <button
        ng-click="toggleDisplay(field)"
        ng-class="::field.display ? 'kuiButton--danger' : 'kuiButton--primary'"
        ng-bind="::field.display ? 'remove' : 'add'"
        class="kuiButton kuiButton--small kuiButton--primary"
        data-test-subj="fieldToggle-{{::field.name}}"
      ></button>
    </div>
  </div>
</li>
```

And then we can also remove this entire selector from main.less:

```less
  ul.discover-selected-fields {
    /* If only _source is presend in the selected fields list, hide its toggle button */
    li:first-child:nth-last-child(1)[attr-field=_source] .discover-field-toggle {
      display: none;
    }
  }
```